demos/main/src/main/java/androidx/media3/demo/main/PlayerActivity.java

```diff
@@ -293,6 +293,9 @@ protected boolean initializePlayer() {
     }
     player.setMediaItems(mediaItems, /* resetPosition= */ !haveStartPosition);
     player.prepare();
+    if (getIntent().getBooleanExtra(IntentUtil.LOOP_PLAYBACK_EXTRA, false)) {
```

Comment:
Mind making a `REPEAT_MODE_EXTRA` and a `REPEAT_MODE_EXTRA_VALUE_NONE`, `REPEAT_MODE_EXTRA_VALUE_ONE`, `REPEAT_MODE_EXTRA_VALUE_ALL` so we cover the entire repeat mode API?

I understand the idea is to start the app directly into the `PlayerActivity` from the command line. We have documented the Intent arguments we support here: https://developer.android.com/media/media3/exoplayer/demo-application#3-firing

I will document this on DAC once we have landed that change. The aim is to have an addition like the following

--es repeat_mode NONE | ONE | ALL

WDYT?

See https://gist.github.com/tsohr/5711945#file-gistfile1-txt-L84